### PR TITLE
some tests for fileIO

### PR DESCRIPTION
**Pull request by sajetan**
PR to know if this works the way I expected.

### CODE REVIEW

File: tests/test_fileIO.py [hunk 1]
```diff
@@ -1,8 +1,9 @@
 from pyhdx import PeptideMasterTable, read_dynamx, KineticsSeries
-from pyhdx.fileIO import txt_to_protein, txt_to_np, read_dynamx
+from pyhdx.fileIO import txt_to_protein, txt_to_np, read_dynamx, fmt_export, _get_f_width
```
Comment (Jhsmit) [inline tests/test_fileIO.py:2]: `_get_f_width` is imported but not used

File: tests/test_fileIO.py [hunk 2]
```diff
@@ -27,4 +28,27 @@ def test_read_dynamx(self):
...
+    def test_methods(self):
+        data = read_dynamx(self.fpath)
+        path = directory / 'test_data' / 'ecSecB_guess.txt';
+
+        #testing fmt_export
+        with pytest.raises(ValueError):
+            fmt_export(data, delimiter=',', width="foo")
+
+        #testing txt_to_protein
+        txt_to_protein(path)
```
Comment (Jhsmit) [inline tests/test_fileIO.py:48]: This is good but you should include an `assert` when you add a function for testing. Here the `txt_to_protein` returns an object, so you could write an assert to check if this is the correct object, and then another which checks if the data in the object is as expected

File: tests/test_fileIO.py [hunk 3]
```diff
@@ -27,4 +28,27 @@ def test_read_dynamx(self):
...
+
+        #testing txt_to_np
+        with open(path, mode='r') as f:
+            txt_to_np(StringIO('f'))
```
Comment (Jhsmit) [inline tests/test_fileIO.py:52]: Same for this function

`array = txt_to_np(...)`
then 
assert array[some selection] == known_value

**Review by Jhsmit — CHANGES_REQUESTED** [review]

**Comment by sajetan** [comment]
reviewed and added some more changes

**Review by Jhsmit — APPROVED** [review]

**Pull request merged**